circle.yml

```diff
@@ -4,3 +4,13 @@ dependencies:
 test:
   override:
     - yarn test
+
+deployment:
+  release:
+    tag: /v[0-9]+(\.[0-9]+)*/
+    commands:
+        - /bin/bash publishTag codesplain-ui
```

Comment:
This should work as simply `./publish` because the shebang identifies the correct interpreter.